src/main/java/io/codelex/flightplanner/model/Flight.java

```diff
@@ -1,16 +1,29 @@
 package io.codelex.flightplanner.model;
 
 import com.fasterxml.jackson.annotation.JsonFormat;
+import lombok.AllArgsConstructor;
 import lombok.Builder;
 import lombok.Data;
+import lombok.NoArgsConstructor;
 
+import javax.persistence.*;
 import java.time.LocalDateTime;
 
+@Entity
+@Table(name = "flights")
+@NoArgsConstructor
+@AllArgsConstructor
 @Data
 @Builder(toBuilder = true)
```

Comment:
paŗāk daudz to anotāciju :smile: 
un iekš @Data jau viss tas ir un kā jau tai citā vietā minēju, šis laikam īšti nebūs builder gadījums
par factory paternu zini? :smile: pats to izmantoju, atstāju tikai `NoArgsConstructor(access = PACKAGE)` un setteriem arī tādu pašu acces un tad šai paša package veidoju `FlightFactory` klasi
taspats arī uz Airport klasi

Reply:
@Data - izveido tikai RequiredArgsConstructor, jeb tikai ar final laukiem
@NoArgsConstructor - nepieciešams JPA
@AllArgsConstructor - hmm, šobrdīd neatminos kam tas bija nepieciešams
@Builder(toBuilder = true) - nepieciešams Unit Testiem

Esmu manījis, ka vietām izmanto 3 klases - AddRequest -> RequestToEntry -> Entry, būs dziļāk jāpapēta šāds piegājiens.